tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,449 @@
+---
+title: An Introduction to Geospatial Analysis
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-10-09
+---
+
+[Geometries - Open In Code Editor](https://code.earthengine.google.com/9d0bd55b88c3588dd189c3b512378a0b)
+
+[Features - Open In Code Editor](https://code.earthengine.google.com/bb1980ff4b6ddab0262de00956f89dcb)
+
+[Images - Open In Code Editor - Geometries](https://code.earthengine.google.com/234ba0ab11dd82112602de05c0bc3bd8)
+
+We will introduce types of geospatial data, using these data on the Earth Engine platforms, and discuss a host of functionality to visualize and analyze them. This workshop was originally created for a workshop during Yale-NUS Data 2.0 hackathon and then updated for Yale GIS Day 2018. 
+
+## Introduction
+Collection, visualization, and analysis of geographical or spatial data.
+
+## Data types
+
+- Vector data represent lat-long coordinates
+
+- Raster data comprises of pixels with associated values
+
+---
+
+
+- Points
+
+![Points](points.png)
+
+- Lines
+
+![Lines](line-vector.png)
+
+- Polygons
+
+![Polygons](polygon-vector.png)
+
+- Raster layers/bands
+
+![Raster](map-algebra.png)
+
+Image sources: https://gisgeography.com/spatial-data-types-vector-raster/
+
+---
+
+## Google Earth Engine platform
+
+[Introductory video](https://www.youtube.com/watch?v=gKGOeTFHnKY&feature=youtu.be, "Video")
+
+[Code Editor](https://code.earthengine.google.com/ "Earth Engine Code Editor")
+
+- Cloud-based platform for planetary scale geospatial analysis
+- Uses Google's computational resources to reduce processing time
+- Massive archive of remote sensing data 
+- 200+ public datasets 
+- greater than 4000 new images every day 
+- greater than 5 million images 
+- greater than 5 petabytes of data
+
+Source: Google Earth Engine User summit
+
+---
+
+![Code Editor (Source: developers.google.com)](ee-editor.png)
+
+---
+
+## Basic Functions
+
+### Declaring variables
+```javascript
+var variableName = ee.ContainerType(value); 
+```
+A container (in the form ee.variabletype) is used to wrap up a native javascript object type so that Google's server can recognize its structure and perform operations on it
+
+###  Centering map
+```javascript
+Map.setCenter(long, lat, zoom level);
+```
+>Zoom level varies from 0 (no zoom) to 20 (highest zoom level)
+
+### Displaying metadata
+```javascript
+print(variableName)
+```
+You cannot print more than 5000 elements at once
```

Comment:
Please add final "." here and throughout.

Reply:
Done (I think?)